src/EventListener/InitializeSystemListener.php

```diff
@@ -275,29 +274,32 @@ class_alias('Contao\\ModuleLoader', 'ModuleLoader');
      */
     private function setRelativePath($basePath)
     {
-        Environment::set('path', $basePath);
```

Comment:
This must not be removed! It will break the `Environment` class, which relies on `path` being set during system initialization.